### PR TITLE
added call to central to save user data

### PR DESCRIPTION
**Pull request by pascalpp**
Fixes https://github.com/StayWokeOrg/intake/issues/4

created a saveUser method which handles the API call, accepting a user and a source, so we can call that method from the web submit endpoint or the SMS API endpoints.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -56,6 +56,7 @@
                         <label for="phone_number">Phone number</label>
                         <input class="form-control input-lg" type="phone" name="phone_number" id="phone_number">
                     </div>
+                    <input type="hidden" name="campaign" id="campaign" value="some_campaign">
```
Comment (mattstauffer) [inline public/index.html:59]: Could we hard code this in Express instead so the user can't override by viewing source?

**Review by mattstauffer — CHANGES_REQUESTED** [review]
This is amazing--so fast to get this out. Just one little note.

File: public/index.html (hunk 1 above)
Comment (pascalpp) [reply public/index.html:59]: Was talking to deray about how they wanna do bucketing. One possibility we discussed was hosting this form at special event- or campaign-specific URLs. e.g. `staywoke.org/in/inauguration` and then we'd consume that last param as the campaign name. whatever form it takes, i think the client should be able to set the campaign. (similarly, the SMS app could accept the campaign as the initial contact keyword, e.g. "text 'inauguration' to 123-456-7890").

so maybe we address this concern by having a whitelist of valid campaigns on the backend. but maybe we should accept any invalid campaign, prefix it with 'unknown:', and then do some data cleanup to address common typos and such.

Comment (mattstauffer) [reply public/index.html:59]: Totally makes sense

**Review by mattstauffer — APPROVED** [review]

**Comment by pascalpp** [comment]
@mattstauffer can you give this another review?

**Review by mattstauffer — APPROVED** [review]
Nice work!!

**Pull request merged**